azure-kusto-data/azure/kusto/data/__init__.py

```diff
@@ -1,4 +1,148 @@
-from .kusto_client import KustoClient, KustoResultIter, KustoResponse
-from .kusto_exceptions import KustoError, KustoClientError, KustoServiceError
+from enum import Enum
+from six import text_type
 from .version import VERSION
 __version__ = VERSION
+
+class KustoConnectionStringBuilder(dict):
+    class _Keywords(Enum):
```

Comment:
No need to have this embedded in the class; the leading `_` is enough to signify that no one else should use this.

Reply:
This class represents the keys of KustoConnectionStringBuilder. This is why I chose to locate _Keywords inside KustoConnectionStringBuilder.